__tests__/app.test.js

```diff
@@ -0,0 +1,89 @@
+const app = require("../db/app")
+const request = require("supertest")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data/")
+const db = require("../db/connection")
+require("jest-sorted")
+
+beforeEach(() => {
+    return seed(data);
+  });
+  afterAll(() => {
+    return db.end();
+  });
+
+describe("GET /api/topics",()=>{
+    test("responds 200 status and all topics with its data",()=>{
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) =>{
+            expect(body.topics.length).toBe(3)
+            body.topics.forEach( topic =>{
+                expect(topic).toMatchObject({
+                    description: expect.any(String),
+                    slug: expect.any(String)
+                })
+            })
+        })
+    })
+    test("responds with 404 with an invalid API request", () => {
+        return request(app)
+          .get("/api/tops")
+          .expect(404)
+          .then((data) => {
+            console.log(data)
+            expect(data.res.statusMessage).toBe("Not Found");
+          });
+      });
+})  
+describe.skip("GET /api",()=>{
```

Comment:
We can't approve a pull request with .skips/.onlys in the test file - otherwise we don't know that your code works

Reply:
I imagine you've added this .skip as this test is failing at the moment - you aren't returning anything from your /api endpoint